script/tool/lib/src/format_command.dart

```diff
@@ -59,13 +59,11 @@ class FormatCommand extends PackageLoopingCommand {
         help: 'Format Kotlin files', defaultsTo: true);
     argParser.addFlag(_javaArg, help: 'Format Java files', defaultsTo: true);
     argParser.addFlag(_swiftArg,
-        help: 'Format and lint Swift files', defaultsTo: true);
+        help: 'Format and lint Swift files', defaultsTo: platform.isMacOS);
```

Comment:
fyi: Nothing to do for this patch, but it looks like swift-format is open source https://github.com/swiftlang/swift-format and tagged for various releases if we one did want to support this on Linux as well.